test/testenv/testenv.go

```diff
@@ -0,0 +1,82 @@
+package testenv
```

Comment:
Should the directory structure be called test/framework/env.go

Reply:
My thought is make 'testenv' the framework name. 'framework' is too general for a package name following guide here https://golang.org/doc/effective_go.html#package-names

Reply:
@douglas-reid since he is good with conventions.